extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -605,7 +605,8 @@ export class AmpAnalytics extends AMP.BaseElement {
       to = {};
     }
 
-    user().assert(opt_predefinedConfig || !from || !from['optout'],
+    user().assert(opt_predefinedConfig || !from || !from['optout'] ||
+                  from['optout'] == '_gaUserPrefs.ioo',
```

Comment:
Nit: indent at +4

Please document why this exception and that we are happy to add more such exceptions.